src/Listeners/RequestTookLongerThanMaxDurationListener.php

```diff
@@ -4,16 +4,27 @@
 
 use Illuminate\Support\Facades\Notification;
 use Taecontrol\MoonGuard\Repositories\UserRepository;
+use Taecontrol\MoonGuard\Notifications\SlackNotifiable;
 use Taecontrol\MoonGuard\Events\RequestTookLongerThanMaxDurationEvent;
 use Taecontrol\MoonGuard\Notifications\RequestTookLongerThanMaxDurationNotification;
 
 class RequestTookLongerThanMaxDurationListener
 {
     public function handle(RequestTookLongerThanMaxDurationEvent $event): void
     {
-        Notification::send(
-            UserRepository::all(),
-            new RequestTookLongerThanMaxDurationNotification($event->uptimeCheck, $event->maxRequestDuration)
-        );
+        $channels = config('moonguard.notifications.channels');
+
+        foreach ($channels as $channel) {
+            $users = ($channel === 'slack') ? new SlackNotifiable() : UserRepository::all();
```

Comment:
Rename this variable into:

```suggestion
            $notifiables = ($channel === 'slack') ? new SlackNotifiable() : UserRepository::all();
```